airflow/providers/presto/transfers/presto_to_slack.py

```diff
@@ -47,11 +43,12 @@ class PrestoToSlackOperator(BaseOperator):
         You can use the default JINJA variable {{ results_df }} to access the pandas dataframe containing the
         SQL results
     :param presto_conn_id: destination presto connection
-    :param slack_conn_id: The connection id for Slack
+    :param slack_conn_id: The connection id for Slack. Mutually exclusive with 'slack_token'
     :param results_df_name: The name of the JINJA template's dataframe variable, default is 'results_df'
     :param parameters: The parameters to pass to the SQL query
     :param slack_token: The token to use to authenticate to Slack. If this is not provided, the
-        'webhook_token' attribute needs to be specified in the 'Extra' JSON field against the slack_conn_id
+        'webhook_token' attribute needs to be specified in the 'Extra' JSON field against the slack_conn_id.py
+        Mutually exclusive with 'slack_conn_id'
```

Comment:
`webhook_token` in Extra marked as Deprecated

https://github.com/apache/airflow/blob/b678dc25f6604ddf603e42500e3e5583b6663df6/airflow/providers/slack/hooks/slack_webhook.py#L103-L111